lib/noether/maybe.ex

```diff
@@ -9,10 +9,10 @@ defmodule Noether.Maybe do
 
   ## Examples
 
-      iex> map(nil, &Kernel.abs/1)
+      iex> Noether.Maybe.map(nil, &Kernel.abs/1)
```

Comment:
The library that we use for Property Testing has a function called `map`, therefore I need to break the ambiguity.